composer.json

```diff
@@ -24,5 +24,9 @@
     "require-dev": {
         "symplify/easy-coding-standard": "dev-main",
         "phpunit/phpunit": "^9.5"
+    },
+    "scripts": {
+        "check-cs": "vendor/bin/ecs check --ansi",
+        "fix-cs": "vendor/bin/ecs check --fix --ansi"
     }
```

Comment:
Instead of the configuration options with paths and parameters, we can run just:

```bash
composer check-cs
```

And the coding standard always run the same :+1: